.github/workflows/build.yml

```diff
@@ -73,11 +73,14 @@ jobs:
           - name: irc
             config: --enable-middle-end=flambda2
             os: ubuntu-latest
-            ocamlparam: ''
+            ocamlparam: '_,w=-46,regalloc=irc,regalloc-param=IRC_SPLIT:off,regalloc-param=IRC_SPILLING_HEURISTICS:flat-uses'
+            check_arch: true
+
+          - name: ls
+            config: --enable-middle-end=flambda2
+            os: ubuntu-latest
+            ocamlparam: '_,w=-46,regalloc=ls,regalloc-param=LS_ORDER:layout'
```

Comment:
I don't know how costly an extra target with flambda2 is, but considering all the other testing LS and IRC have already had, I think it is enough to just have one, that sets `regalloc=cfg` as per the comment in `asmgen`.   LS will get tested on entry functions.

Reply:
Sounds good.